crates/stateless/src/fork_spec.rs

```diff
@@ -91,5 +94,13 @@ impl From<ForkSpec> for ChainSpec {
             ForkSpec::Prague => spec_builder.prague_activated(),
         }
         .build()
+        .hardforks;
+
+        Self {
+            chain: Chain::mainnet(),
+            hardforks,
+            deposit_contract_address: Some(MAINNET_DEPOSIT_CONTRACT_ADDRESS),
+            blob_params: BlobScheduleBlobParams::default(),
```

Comment:
Since this `ForkSpec` is a mirror of the ef-tests, its correct to assume mainnet configs since that's what EEST expects.